src/Checker.elm

```diff
@@ -0,0 +1,62 @@
+module Checker exposing (getIssues, getIssuesJSON, getIssuesString)
+
+import Configuration
+import Constraints
+import Encoders
+import Issue
+import Json.Encode
+import Models
+
+
+getIssues : Models.ParsedModule -> Configuration.Model -> List Issue.Issue
+getIssues parsed config =
+    let
+        ignoredChecks =
+            config.excludedChecks
+
+        checkAll =
+            config.checkAllDefinitions
+
+        topLevelLine =
+            parsed.topLevelComment
+                |> Maybe.map (\( range, _ ) -> range.start.row)
+                |> Maybe.withDefault 1
+
+        topLevelCommentIssues =
+            Constraints.getViolationsTopLevel parsed.topLevelComment ignoredChecks
+                |> (\violated -> Issue.fromViolationsAndTrigger violated (Issue.TopLevel parsed.topLevelComment))
+                |> (\issue -> [ ( topLevelLine, issue ) ])
+
+        otherCommentsIssues =
+            parsed.otherComments
+                |> List.map (\comment -> ( comment, Constraints.getViolationsDangling comment ignoredChecks ))
+                |> List.map (\( comment, violated ) -> ( comment, Issue.fromViolationsAndTrigger violated (Issue.Dangling comment) ))
```

Comment:
Is there a policy for max line length in Elm? It would be good to enforce it here at 80 chars for easier Github reviews etc.